rust/ql/test/extractor-tests/generated/ArgList/ArgList.ql

```diff
@@ -2,9 +2,8 @@
 import codeql.rust.elements
 import TestUtils
 
-from ArgList x, int getNumberOfArgs
-where
-  toBeTested(x) and
-  not x.isUnknown() and
-  getNumberOfArgs = x.getNumberOfArgs()
```

Comment:
The new test does not output `getNumberOfArgs`; is that deliberate?

Reply:
yes, as the entire list of args is in the same expected file, so I think that information is redundant, and reducing the number of columns helps readability

Reply:
same goes for `has...` for optional properties